tests/playground.html

```diff
@@ -2,12 +2,34 @@
 
 <head>
     <title>Button Counter</title>
+    <script src="http://code.jquery.com/jquery-latest.min.js"></script>
```

Comment:
Make sure you are branching from main to a new branch and not from the other PR branch.

Reply:
looks like it is now corrected. And this branch doesn't contain any changes in this file.